src/test/java/org/prebid/server/ApplicationTest.java

```diff
@@ -268,6 +268,74 @@ public void ampShouldReturnTargeting() throws IOException {
                 .body(Matchers.equalTo(jsonFrom("amp/test-amp-response.json")));
     }
 
+    @Test
```

Comment:
Do we really need these 2 tests in ApplicationTest? m.b. it is possible to add params to `ampShouldReturnTargeting()` test.